Casks/p/plasticscm-cloud-edition.rb

```diff
@@ -9,8 +9,8 @@
   homepage "https://www.plasticscm.com/"
 
   livecheck do
-    url "https://www.plasticscm.com/download/releasenotes/"
-    regex(/plastic\sscm\s-\srelease\snotes\s-\s(\d+(?:\.\d+)+)/i)
+    url "https://www.plasticscm.com/download"
+    regex(%r{href=.*?/download/v?(\d+(?:\.\d+)+)/}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/download/v?(\d+(?:\.\d+)+)/plasticscm/[^/]+/cloudedition}i)
```

There may be something to be said for specifically matching the URL for the cloud edition, as a less specific regex would also match the URL for the full installer on the download page (e.g., `/download/11.0.16.9213/plasticscm/macosx/full`). These versions are the same at the moment and will probably stay that way but better safe than sorry.